src/main/kotlin/ch/hsr/dsl/dwrtc/websocket/Main.kt

```diff
@@ -12,7 +16,15 @@ fun main(args: Array<String>) {
             // or .enableStaticFiles("/public") to use Class Path
             .start(config[WEBSERVER_PORT])
 
-    val service = ClientService()
+    val service = try {
```

Comment:
Hope this is clear... If not, we might add a comment (or convert to a non-exception driven style, "exceptions should be exceptional")